beacon-light-client/solidity/contracts/validators_accumulator/ValidatorsAccumulator.sol

```diff
@@ -15,10 +15,9 @@ contract ValidatorsAccumulator is IValidatorsAccumulator {
 
   // A counter for the total number of validators
   uint256 internal validatorsCount;
-  // Start index of validators map
-  uint256 internal startIndex;
```

Comment:
We should do that in another PR as it was reported as separate issue by audtiors